### DISCUSSION TITLE
Improve the reconstructed source of DESIJ1554+0443

### DISCUSSION THREAD

**Discussion by ZareefJafar** [AstroBridge/BDLensing#48] [Q&A] [answered]
I am working with the DESIJ1554+0443 data. The main issue with the image is that the lensed light of the source galaxy is really faint and really close to the lens galaxy. Although my reduced chi-square value is 1.041, the "Reconstructed source" does not look right. If you see the smaller circular mask nearest to the center, it is maybe on a lensed source arc. Do not know if it is the right move or not. Here is my [notebook](https://github.com/ZareefJafar/BDLensing/blob/main/DESIJ1554%2B0443/model_DESIJ1554%2B0443.ipynb) 

Thanks,
@ZareefJafar    

![Screenshot from 2023-04-30 05-24-24](https://user-images.githubusercontent.com/49613484/235328126-7dc7727e-e4c9-440d-ae7b-c137038f0da1.png)

**Comment by ajshajib** [comment, marked as answer]
Try setting an upper limit for the source size as suggested here: https://github.com/ajshajib/BDLensing/discussions/38#discussioncomment-5698159.
